### DISCUSSION TITLE
Change default ka= algorithm

### DISCUSSION THREAD

**Discussion by hackerfactor** [hackerfactor/SEAL#5] [Ideas]
As mentioned in #4, RSA is going to be deprecated as soon as quantum crypto comes out. Defaulting to "ka=rsa" means VIDA will default to a "soon to be obsolete" crypto algorithm.

The recommendation from named-dodo is to not have a default algorithm. Make "ka=" a required parameter, with "ka=rsa" being initially supported (until RSA becomes deprecated.).

**Comment by named-dodo** [comment]
Actually, to be more precise, RSA will be deprecated the moment we get quantum computers to do the first realistic prime factorization.
This may take 20 to 50 years but it's also possible that a breakthrough is made and it happens in less than 5 years. 😰

I simply wouldn't want to see this protocol be defaulting to a broken algorithm.
I'm fine with the default digest algorithm sha256, because as far as i am aware that one will be just fine in the future.

**Comment by hackerfactor** [comment]
Merge accepted.
